app/components/common/eventCard/index.tsx

```diff
@@ -0,0 +1,39 @@
+interface EventCardProps {
+  date: number;
```

Comment:
Wouldn't this be Date format?

Reply:
+1

Reply:
Sure, I am updating on it 
Thanks!

Reply:
Are we sure we will get exact formatted date from the backend?

Reply:
this is still not addressed. Please refer the event object that we will receive from backend and update accordingly.

Reply:
On call, as we decided to create a separate ticket when we have `days.js` in our codebase. 
As of now I am hard coding the `date, month and time`  

Cc: @iRohitGaur @DeRaowl @harshith-venkatesh